src/lib.rs

```diff
@@ -571,14 +630,14 @@ impl GooseAttack {
                 if u == 0 {
                     if self.configuration.worker {
                         error!("At least 1 user is required.");
-                        std::process::exit(1);
+                        return Err(GooseError::InvalidOption);
                     } else {
                         0
                     }
                 } else {
                     if self.configuration.worker {
                         error!("The --users option is only available to the manager.");
-                        std::process::exit(1);
+                        return Err(GooseError::InvalidOption);
```

Comment:
Would it make sense to include the string in the GooseError::InvalidOption in case someone does not want to display it via error!()? [can be follow-up as it's unrelated]

Reply:
I have not explored how to put more data into the Error, and decided to explore this as a followup as the PR is already so large. But I agree, it would be nice to include more information, and to not force an `error!()` on them.